homeassistant/components/freebox/__init__.py

```diff
@@ -72,8 +90,6 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Unload a config entry."""
     unload_ok = await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
     if unload_ok:
-        router = hass.data[DOMAIN].pop(entry.unique_id)
-        await router.close()
-        hass.services.async_remove(DOMAIN, SERVICE_REBOOT)
+        hass.data[DOMAIN].pop(entry.unique_id)
```

Comment:
The closing of the api and the removal of the service have been moved to `entry.async_on_unload`